components/alert/f-alert.vue

```diff
@@ -33,15 +33,18 @@ export default { name: 'fAlert' }
 import { computed } from 'vue'
 import { fExpandTransition } from '#generics'
 import { createModel, modelProps } from 'create-v-model'
-import { colorMap, possibleColorBooleans, colorBooleanProps } from './logic.js'
+import { colorMap, possibleColorBooleans } from './logic.js'
 
 const props = defineProps({
   title: String,
   role: {
     type: String,
     default: 'alert'
   },
-  ...colorBooleanProps,
```

Comment:
This is already fixed in Warp - but this was causing the Alert component to be included in builds.